### DISCUSSION TITLE
紅綠燈搭配AI

### DISCUSSION THREAD

**Discussion by 109302013evantsao** [1101-GS4719-C/Python-Community-GS4719-C#68] [General]
有了AI的計算和幫助 能夠減少油耗亦可以減少時間的消耗 不用再沒車的路上等90秒了
[紅綠燈](https://chinese.engadget.com/google-ai-traffic-lights-research-environment-pollution-070132356-103028835.html)

**Comment by HSU-YU-JEN** [comment]
這個技術好棒 希望可以引進台灣

**Comment by chun1203** [comment]
用AI 導入到紅綠燈感覺對台灣上下班交通壅塞的狀況能有很大的幫助

**Comment by 109302003** [comment]
感覺用AI計算紅綠燈配合自駕車也能讓台灣的交通更順暢

**Comment by Jie2222** [comment]
好像在台灣某些地方有類似概念的紅綠燈控制，感覺是個可行性高，也相對容易普及的技術

**Comment by chia-kai-chang** [comment]
**節能減碳 永續地球** 是世界趨勢，基於這樣的趨勢而開發的AI技術，相對容易被普及應用。

**Comment by 20010906** [comment]
希望此技術可以繼續研究開發，並實際應用於未來交通

**Comment by zixunn** [comment]
可以減少汽機車等待紅綠燈的油耗量以及排氣量，同時兼顧減少空氣污染以及舒緩交通壅塞的功能，感覺這個技術非常實用，好期待能在台灣被應用。
